src/ceph-volume/ceph_volume/devices/raw/common.py

```diff
@@ -26,20 +26,23 @@ def create_parser(prog, description):
         dest='crush_device_class',
         help='Crush device class to assign this OSD to',
     )
-    parser.add_argument(
-        '--osd-id',
-        help='Reuse an existing OSD id',
-    )
-    parser.add_argument(
-        '--osd-fsid',
-        help='Reuse an existing OSD UUID',
-    )
```

Comment:
Why remove these?

Reply:
We don't use them and I'm trying to keep this implementation as light as possible. If we decide to use this later we can easily re-add them.